### PR TITLE
ARUHA-305: change de.zalando.aruha to org.zalando

### PR DESCRIPTION
**Pull request by v1ctor**

### CODE REVIEW

**Comment by adyach** [comment]
This change removed all static imports 👍

File: src/test/java/org/zalando/nakadi/service/subscription/model/PartitionTest.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+import static java.util.Collections.singletonList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+public class PartitionTest {
```
Comment (v-stepanov) [inline src/test/java/org/zalando/nakadi/service/subscription/model/PartitionTest.java]: why this file is completely green?

**Comment by v-stepanov** [comment]
I don't like all these changes about static imports.
It creates a huge amount of diff noise. We will have hard times checking the Git history of some affected files.

**Comment by v1ctor** [comment]
Revert this.

**Comment by adyach** [comment]
:shipit:

**Pull request merged**